docs/API.md

```diff
@@ -1152,3 +1152,87 @@ Response:
 	]
 }
 ```
+
+
+### V2 System Utilities
+
+#### Journald logs
+
+Added in supervisor version v9.18.0
+
+Get access to the journald logs from the device. The logs
+are parsed to extract certain information, and the original 
+log is included too. The original log format can be found
+[here](https://www.freedesktop.org/software/systemd/man/journalctl.html#%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20json%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20).
+The fields added by the supervisor are:
+```json
+{
+	message: "A string representation of the message",
```

Comment:
Keys need to be surrounded by double quotes.
Or just change `json` to `typescript`.

Reply:
I actually changed that, must've forgot to push the changed version...